keps/sig-network/1435-mixed-protocol-lb/README.md

```diff
@@ -476,6 +476,20 @@ Kube-proxy will not block traffic based on the port status information from `Ser
 
 ### Test Plan
 
+[x] I/we understand the owners of the involved components may require updates to
+existing tests to make this code solid enough prior to committing the changes necessary
+to implement this enhancement.
+
+##### Prerequisite testing updates
+
+##### Unit tests
+
+Proper unit tests are in place for the affected code parts.
+
+##### Integration tests
```

Comment:
For GA I think it would be reasonable to have an integration test for this. The test doesn't have to actually create an LB, but just prove that admission of a Service with mixed protocol is allowed in the API

Reply:
For the time being I have added the need for integration tests. I could not provide a link as those do not exist yet,

Reply:
I disagree @andrewsykim, I think that the e2e test should exist and be skipped on the CPIs that doesn't implement it

Reply:
there is also a service.Status.Condition that indicates that the feature is not implemented, I think the e2e test is useful for the ecosystem and users

Reply:
Definitely wouldn't object to have more tests here. But given the feature itself is an API-only change relaxing some validation around ports, I don't think it's fair to require an e2e test that will rely on one of the providers to test it. (this includes the condition) 

You have a good point about kube-proxy behavior though, maybe a kube-proxy specific test that just validates it can work with mixed protocols might be good (though I would hope we have this already)

Reply:
Maybe this is a reasonable ask:

An e2e test in Kind, that receives a Service Type=LB w/ mixed protocol and verifies kube-proxy data path in-cluster, but doesn't actually require an LB to be created.